engines/v8/src/link.cc

```diff
@@ -0,0 +1,116 @@
+
+#include "link.hh"
+#include <iostream>
+#include <map>
+#include "wasi.hh"
+
+std::vector<ImportName> list_imports(const wasm::Module* module) {
+  auto import_types = module->imports();
+  auto import_names = std::vector<ImportName>();
+  for (size_t i = 0; i < import_types.size(); ++i) {
+    wasm::ImportType* type = import_types[i].get();
+    auto module_str = std::string(type->module().get(), type->module().size());
+    auto name_str = std::string(type->name().get(), type->name().size());
+    // std::cerr << "Found import: " << module_str << " " << name_str <<
+    // std::endl;
+    import_names.push_back(ImportName(module_str, name_str));
+  }
+  // std::cerr << "Number of imports: " << import_names.size() << std::endl;
```

Comment:
nitpick: delete this commented out logging or add a proper logging macro that can turn logging on/off at build time or via env var.